lib/minesweeper_2pl/board.rb

```diff
@@ -1,4 +1,31 @@
 module Minesweeper_2pl
+
   class Board
+    attr_accessor :size, :bomb_count, :bomb_positions, :positions, :row_size
+
+    def set_positions(positions)
+      set_row_size(positions)
+      set_bombs
+      @positions = []
+      positions.times do |position|
```

Comment:
The parameter named `positions` (an integer) and the instance variable `positions` (an array) are pretty confusing here. Is there a better name for one or the other that can clarify the meaning?